gem/templates/springster/base.html

```diff
@@ -61,6 +65,40 @@
       <link rel="stylesheet" type="text/css" href="{{ STATIC_URL }}css/{{ ENV }}/05_no-script-state.css" media="all">
     {% endcompress %}
     </noscript>
+    <script type="text/javascript">
+      let imageIdentity = document.getElementById("image__identity"),
+          getImageArray = imageIdentity.src.split("."),
+          imageExt = getImageArray.slice(-1),
+          formats = [
+            {
+              type:"image",
+              ext: ["png","jpg","jpeg","gif","svg"]
+            },
+            {
+              type:"media",
+              ext: ["mpeg","ogg","jpeg"]
+            }
+          ];
+          for(let i = 0; i < formats.length; i++){
+            if(formats[i].type == "image") {
+              console.log(formats[i].type);
+              for(let j = 0; j < formats[i].ext.length; j++) {
+                //formats[i].ext.includes(imageExt)
+                if(formats[i].ext[j] == imageExt) {
+                  console.log(document.querySelector("meta[property='og:type']"));
+                  document.querySelector("meta[property='og:type']").setAttribute("content",formats[0].type + "/" + formats[0].ext[i]);
+                }
+              }
+            } else if(formats[i].type == "media") {
+              for(let j = 0; j < formats[i].ext.length; j++) {
+                if(formats[i].ext[j] == imageExt) {
+                  document.querySelector("meta[property='og:type']").setAttribute("content",formats[0].type + "/" + formats[0].ext[i]);
+                }
+              }
+            }
+          }
+
+    </script>
   </head>
```

Comment:
@Mitso I can create a django template filter for this, seeing that we are having more and more media content that might require it
```
import mimetypes    

>>> mimetypes.guess_type('a.py',strict = True)
('text/x-python', None)
```